native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/DetailActivity.java

```diff
@@ -93,9 +93,9 @@ public void onResume(RestClient client) {
 	}
 
 	@Override
-	public void onDestroy() {
+	public void onPause() {
```

Comment:
We were registering the loader in `onResume`, so ideally we should destroy it in `onPause`. We don't want a `Loader` hanging around once the screen goes out of scope.